libs/application/templates/driving-instructor-registrations/src/fields/ViewStudent/index.tsx

```diff
@@ -195,55 +249,110 @@ const ViewStudent = ({
               <Text variant="h4">{formatMessage(m.viewStudentName)}</Text>
               <Text variant="default">{student.name}</Text>
             </GridColumn>
-            <GridColumn span={['12/12', '4/12']} paddingBottom={[3, 0]}>
+            {/* nationalId */}
+            <GridColumn span={['12/12', '6/12']} paddingBottom={[3, 0]}>
               <Text variant="h4">{formatMessage(m.viewStudentNationalId)}</Text>
-              <Text variant="default">{student.nationalId}</Text>
+              <Text variant="default">
+                {formatKennitala(student.nationalId)}
+              </Text>
             </GridColumn>
-            <GridColumn span={['12/12', '4/12']} paddingBottom={[3, 0]}>
+          </GridRow>
+          <GridRow marginBottom={3}>
+            {/* Completed */}
+            <GridColumn span={['12/12', '6/12']} paddingBottom={[3, 0]}>
               <Text variant="h4">
                 {formatMessage(m.viewStudentCompleteHours)}
               </Text>
               <Text variant="default">
                 {student.book?.totalLessonCount ?? 0}
               </Text>
             </GridColumn>
+            {/* Practice driving */}
+            <GridColumn span={['12/12', '6/12']} paddingBottom={[3, 0]}>
+              <Text variant="h4">
+                {formatMessage(m.viewStudentPracticeDrivingTitle)}
+              </Text>
+              <Text variant="default">
+                {student.book?.practiceDriving
+                  ? formatMessage(m.viewStudentYes)
+                  : formatMessage(m.viewStudentNo)}
+              </Text>
+            </GridColumn>
           </GridRow>
-
           <GridRow marginBottom={5} className={styles.hideRow}>
+            {/* Completed schools */}
             <GridColumn span={['12/12', '6/12']}>
               <Text variant="h4">
                 {formatMessage(m.viewStudentCompleteSchools)}
               </Text>
-              {student.book?.drivingSchoolExams?.map((school, key) => {
-                const datePostfix = school.examDate
-                  ? `- ${school.examDate}`
-                  : ''
-                return (
-                  <Text key={key} variant="default">
-                    {`${school.schoolTypeName}${datePostfix}`}
-                  </Text>
-                )
-              })}
+              {completedSchools.length > 0 ? (
+                completedSchools?.map((school, key) => {
+                  const datePostfix = school.examDate
+                    ? `- ${format(new Date(school.examDate), 'dd.MM.yyyy')}`
+                    : ''
+                  return (
+                    <Text key={key} variant="default">
+                      {`${school.schoolTypeName}${datePostfix}`}
+                    </Text>
+                  )
+                })
+              ) : (
+                <Text variant="default">
+                  {formatMessage(m.viewStudentNoCompleteSchools)}
```

Comment:
Can this be moved to a pretty-named function or variable? Hard to read the intent of the code.

Reply:
Sure, this is code from you that I moved around but let's make it more readable.